github-action/src/main.ts

```diff
@@ -264,17 +312,74 @@ export async function runPost(): Promise<void> {
 
 	const platform = emptyStringAsUndefined(core.getInput('platform'));
 	if (platform) {
+		// Create a digests object to track digests for each platform
+		const digestsObj: Record<string, string> = {};
+		const platforms = platform.split(/\s*,\s*/);
+		
 		for (const tag of imageTagArray) {
 			core.info(`Copying multiplatform image '${imageName}:${tag}'...`);
 			const imageSource = `oci-archive:/tmp/output.tar:${tag}`;
 			const imageDest = `docker://${imageName}:${tag}`;
 
 			await copyImage(true, imageSource, imageDest);
+			
+			// After pushing, get and set digest
+			const inspectCmd = await exec('docker', ['buildx', 'imagetools', 'inspect', `${imageName}:${tag}`, '--format', '{{json .}}'], { silent: true });
+			if (inspectCmd.exitCode === 0) {
+				try {
+					const imageInfo = JSON.parse(inspectCmd.stdout);
+					
+					// If it's a manifest list, extract digests for each platform
+					if (imageInfo.manifests) {
+						for (const manifest of imageInfo.manifests) {
+							if (manifest.platform && manifest.digest) {
+								const platformStr = `${manifest.platform.os}/${manifest.platform.architecture}${manifest.platform.variant ? '/' + manifest.platform.variant : ''}`;
```

Comment:
I also had to change this line to:

```js
const platformStr = `${manifest.platform.os}/${manifest.platform.architecture}${manifest.platform.variant ? `/${manifest.platform.variant}` : ''}`;
```
because I was getting an eslint error about an `Unexpected string concatenation`.